deploy/charts/cert-manager/templates/rbac.yaml

```diff
@@ -10,14 +10,19 @@ metadata:
     heritage: {{ .Release.Service }}
 rules:
   - apiGroups: ["certmanager.k8s.io"]
-    resources: ["certificates", "certificates/finalizers", "issuers", "clusterissuers", "orders", "orders/finalizers", "challenges"]
+    resources: ["certificates", "certificates/finalizers", "issuers", "issuers/finalizers","clusterissuers", "orders", "orders/finalizers", "challenges",  "challenges/finalizers"]
```

Comment:
We don't use finalizers on Issuer resources - are you sure this permission is required?